module-3/lambda_function.py

```diff
@@ -8,7 +8,7 @@
 import ask_sdk_core.utils as ask_utils
 import os
 from ask_sdk_s3.adapter import S3Adapter
-s3_adapter = S3Adapter(bucket_name=os.environ["S3_PERSISTENCE_BUCKET"])
+s3_adapter = S3Adapter(bucket_name="NAME_OF_YOUR_BUCKET_HERE")
```

Comment:
Why are we removing os.environ from here? That's how an Alexa Hosted skill references an S3 bucket created by Alexa Hosted.

Reply:
Please see [the comment above](https://github.com/alexa/skill-sample-python-first-skill/pull/33#discussion_r480590543)